posthog/models/organization.py

```diff
@@ -113,7 +114,10 @@ class Level(models.IntegerChoices):
 
     class Meta:
         constraints = [
-            models.UniqueConstraint(fields=["organization_id", "user_id"], name="unique_organization_membership")
+            models.UniqueConstraint(fields=["organization_id", "user_id"], name="unique_organization_membership"),
+            models.UniqueConstraint(
+                fields=["organization_id"], condition=models.Q(level=15), name="only_one_owner_per_organization"
```

Comment:
Why?

Reply:
This is to ensure there are no multiple owners at the same time. Originally I thought about adding foreign key `owner` to `Organization`, but decided that using the existing `OrganizationMembership` with a constraint will work better.